map/map_tests/user_routes_test.cpp

```diff
@@ -0,0 +1,42 @@
+#include "testing/testing.hpp"
+
+#include "map/routing_manager.hpp"
+
+#include <string>
+#include <vector>
+
+namespace user_routes_test
+{
+using namespace std;
+
+#define RM_CALLBACKS {                                                           \
+     static_cast<RoutingManager::Callbacks::DataSourceGetterFn>(nullptr),        \
+     static_cast<RoutingManager::Callbacks::CountryInfoGetterFn>(nullptr),       \
+     static_cast<RoutingManager::Callbacks::CountryParentNameGetterFn>(nullptr), \
+     static_cast<RoutingManager::Callbacks::GetStringsBundleFn>(nullptr),        \
+     static_cast<RoutingManager::Callbacks::PowerManagerGetter>(nullptr)         \
+  }
+
+class TestDelegate : public RoutingManager::Delegate
+{
+  void OnRouteFollow(routing::RouterType type) override
+  {
+    // Empty
+  }
+
+  void RegisterCountryFilesOnRoute(std::shared_ptr<routing::NumMwmIds> ptr) const override
+  {
+    // Empty
+  }
+};
+
+UNIT_TEST(user_routes_test)
```

Comment:
Is this line printed when you run `../omim-build-debug/map_tests` binary?

Reply:
Yes, thanks for pointing me to the binary. This way it shows up. I'll finish up the test in the morning.

Reply:
This test has now been expanded to further tests